apollo-router/tests/fixtures/test.rhai

```diff
@@ -0,0 +1,26 @@
+// This is a test used for rhai plugin
+
+fn map_router_service_response(context) {
```

Comment:
I think we should get rid of `map_` so `map_router_service_response` just becomes `router_service_response`
The `map_` doesn't really aid the user to understand what is going on, and it's really just an underlying implementation detail.

Reply:
Ok I thought it was more clear to have this `map` naming because we are taking the context, mutate it and return it so basically it's a map. Ok will do the change